### DISCUSSION TITLE
Manage Supabase credentials

### DISCUSSION THREAD

**Discussion by TannerGabriel** [supabase/supabase#5674] [Questions]
I'm currently trying to self-host Supabase on my server and got a basic setup working. That said, it is pretty bothersome to create all the needed credentials in multiple places (e.g. `.env`, `kong.yaml`, etc.).

Is there some way you could automate the process e.g. automatically generating a secret at the beginning and using it in all those files?

**Comment by nicholasburka** [comment]
don't know if you still could use an answer to this question; sounds like you could use a secret manager (noted in the bottom of https://supabase.com/docs/guides/hosting/overview#configuration). i'm planning to do this too, so if you've already figured this out i'd appreciate any notes/tips, otherwise i'd be glad to share once i have
